test/unit/test_event_props_reshaper.py

```diff
@@ -215,22 +189,21 @@ def test_should_reshape_whole_objects():
         }
     }
     schema = ReshapeSchema(
-        condition="",
         template={
             "all": "event@..."
         }
     )
 
-    EventPropsReshaper(
-        event=(event := Event(
+    event = EventPropsReshaper(
+        event=(_event := Event(
```

Comment:
:= pseudo code it not meant to be use in parameter assignment. It make the code very unclear.